WeScan/Common/AdaptiveThresholdingCIFilter.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AdaptiveThresholdingCIFilter.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 13/11/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import CoreImage
+
+final class AdaptiveThresholdCIFilter: CIFilter {
```

Comment:
This class uses a bit of a weird pattern. 
I think that this could be made into a more generic `class` (or `struct`) called something like `ImageProcessing`.

After that, we could simply create a static function which does the adaptive threshold filtering.
For example:
```
static func applyAdaptiveThreshold(image: CIImage) -> CIImage? {
   // Implementation
}
```

Reply:
Fixed in ad8c234